uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -249,11 +248,16 @@ private void validateInstanceToTopicPartitionsMap(
     LOGGER.info("\n\nFor controller instanceToTopicPartitionsMap:");
 
 
-    Map<String, String> instanceIdAndNameMap = HelixUtils.getInstanceToHostnameMap(_helixManager);
+    Map<String, Pair<String, Integer>> instanceIdAndNameMap = HelixUtils.getInstanceToHostInfoMap(_helixManager);
     int validateWrongCount = 0;
     int lowUrgencyValidateWrongCount = 0;
     for (String instanceId : instanceToTopicPartitionsMap.keySet()) {
-      String hostname = instanceIdAndNameMap.containsKey(instanceId) ? instanceIdAndNameMap.get(instanceId) : "";
+      Pair<String, Integer> hostInfo = null;
+      try {
+        hostInfo = getHostInfo(instanceId);
```

Comment:
1. Shall we do `continue;` when after get Exception from `getHostInfo(instanceId)`
2. Please add more information into the error log, example :  
`                   LOGGER.error("Validate WRONG: Get topics error when connecting to {} for route {}", hostname, routeSet, e);
`

Reply:
1. if we do continue after get excepiton , we may miss the route validation.
2. move below log(at line 375) up here to avoid double logging